enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/BackButton.tsx

```diff
@@ -5,7 +5,9 @@ import { useInteractiveQuestionContext } from "../context";
 export const BackButton = () => {
   const { onNavigateBack } = useInteractiveQuestionContext();
 
-  return (
-    onNavigateBack && <DashboardBackButton noLink onClick={onNavigateBack} />
-  );
+  if (!onNavigateBack) {
+    return null;
+  }
```

Comment:
For some reason, this caused the back button to crash in React 17. Is this because the `onNavigate` back line coalesces to non-null values?